frontend/src/components/GroupBar/index.jsx

```diff
@@ -13,26 +15,80 @@ function GroupBar( {id, changeId, page} ) {
     }, []);
 
     const changeToEditMode = () => {
+        if(editMode){
+            setEditGroup({id:-1,name:"",description:""})
+            setGroups(groups.filter((element)=> element.id !== -1))
+        }
         setEditMode(!editMode);
     }
 
-    const changeInformation = () => {
-        
+    const setupCreateGroup = () =>{
+        if(!groups.some((element)=> element.id == -1))
+            setGroups([...groups,{id:-1,name:"",description:""}])
     }
 
+    const changeEditGroupName = (name) =>{
+        setEditGroup({
+            id:editGroup.id,
+            name: name,
+            description: editGroup.description,
+        })
+    }
+    const cahngeEditGroupDescription = (description) =>{
+        setEditGroup({
+            id:editGroup.id,
+            name: editGroup.name,
+            description: description,
+        })
+    }
+    
+    const handleUpdateCreate = () =>{
+        if (editGroup.id < 0){
+            monitorsGroupService.createGroup(editGroup).then((response) =>{
+                monitorsGroupService.getGroups().then((groupsData) => {
+                    setGroups(groupsData.data);
+                })
+            })
+        }else{
```

Comment:
Either keep every `if... else` condition across every file with this aspect or put `else` below the bracket